src/pages/settings/StatCard.tsx

```
@@ -11,10 +11,7 @@ const StatCard = ({ stats }: StatCardProps) => {
     return (
         <div className="bg-background relative z-20 -mt-10 mb-8 flex justify-evenly rounded-2xl py-3 shadow-xl">
             {stats.map((item, index) => (
-                <div
-                    key={index}
-                    className="flex flex-col items-center justify-between"
-                >
+                <div className="flex flex-col items-center justify-between">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)